code/modules/spells/spellbook.dm

```diff
@@ -40,6 +40,11 @@ var/list/artefact_feedback = list(/obj/structure/closet/wizard/armor = 		"HS",
 	name = spellbook.name
 	desc = spellbook.desc
 
+/obj/item/weapon/spellbook/pickup(mob/living/user as mob)

```

Comment:
11. Pretty sure space wizards can be not in the space wizard faction if adminspawned, as well. it'd be better to use is_wizard or whatever the check was here.